### PR TITLE
K8SPG-258 Bundle for cluster-wide operator installation

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Comment by egegunes** [comment]
@cap1984 did you create the bundle manually? this bundle won't be updated if we do any changes in CRD or RBAC. We should use kustomize for this. If you want I can push necessary stuff here

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-307/f7aae6bb/init-deploy.log) |
| We run 1 out of 1|

commit: https://github.com/percona/percona-postgresql-operator/pull/307/commits/f7aae6bb65869005c041c4b4d940f6090d03dd3e
image: `perconalab/percona-postgresql-operator:PR-307-f7aae6bb`

**Comment by cap1984** [comment]
> @cap1984 did you create the bundle manually? this bundle won't be updated if we do any changes in CRD or RBAC. We should use kustomize for this. If you want I can push necessary stuff here

kustomize adjustments have been added

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-307/f875bae0/init-deploy.log) |
| We run 1 out of 1|

commit: https://github.com/percona/percona-postgresql-operator/pull/307/commits/f875bae0bd064e2232d76bd47f8c54171f8c7e4a
image: `perconalab/percona-postgresql-operator:PR-307-f875bae0`

**Comment by hors** [comment]
> > @cap1984 did you create the bundle manually? this bundle won't be updated if we do any changes in CRD or RBAC. We should use kustomize for this. If you want I can push necessary stuff here
> 
> kustomize adjustments have been added

What about cw-rbac.yaml? Should we have it ? Or if rbac.yaml allow us to run operator in CW maybe we need to change/correct it?

**Comment by cap1984** [comment]
> > > @cap1984 did you create the bundle manually? this bundle won't be updated if we do any changes in CRD or RBAC. We should use kustomize for this. If you want I can push necessary stuff here
> > 
> > 
> > kustomize adjustments have been added
> 
> What about cw-rbac.yaml? Should we have it ? Or if rbac.yaml allow us to run operator in CW maybe we need to change/correct it?

I'd rather prefer one file only approach for cw specific case. Explicit cw rbac file requires additional namespace hardcode and it won't be suitable for the end user to operate with.
However I'm open for a discussion.

**Comment by hors** [comment]
> > > > @cap1984 did you create the bundle manually? this bundle won't be updated if we do any changes in CRD or RBAC. We should use kustomize for this. If you want I can push necessary stuff here
> > > 
> > > 
> > > kustomize adjustments have been added
> > 
> > 
> > What about cw-rbac.yaml? Should we have it ? Or if rbac.yaml allow us to run operator in CW maybe we need to change/correct it?
> 
> I'd rather prefer one file only approach for cw specific case. Explicit cw rbac file requires additional namespace hardcode and it won't be suitable for the end user to operate with. However I'm open for a discussion.
it is ok for now

File: config/default/kustomization.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 namespace: postgres-operator
 
 commonLabels:
-  postgres-operator.crunchydata.com/control-plane: postgres-operator
+  pg.percona.com/control-plane: postgres-operator
```
Comment (hors) [inline config/default/kustomization.yaml:4]: @egegunes @cap1984 should we have the same for bundle.yaml ?

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator-v2/PR-307/2c6893f2/init-deploy.log) |
| We run 1 out of 1|

commit: https://github.com/percona/percona-postgresql-operator/pull/307/commits/2c6893f261713bc1ac55103ebb8f49f5ba70797b
image: `perconalab/percona-postgresql-operator:PR-307-2c6893f2`

**Review by hors — APPROVED** [review]

**Pull request merged**